config/checkstyle-input-suppressions.xml

```diff
@@ -68,46 +68,6 @@
           files="[\\/]packageobjectfactory[\\/]zoo[\\/]FooCheck.java"/>
 
   <!-- until https://github.com/checkstyle/checkstyle/issues/13213 -->
-  <suppress id="UnnecessaryOkComment"
```

Comment:
Please look at the failing CI :
https://dev.azure.com/romanivanovjr/romanivanovjr/_build/results?buildId=19900&view=logs&j=c902ebb4-c9f8-5f09-4e17-ff78fbbc842e&t=9ca98c81-ff64-58f0-9d03-a23ac1c4a111&l=1423

`Commit e038f6c4a74190fe8a96b6513d04deebc2687f73 message: "Issue #13213:Removing unnecessary ok comments\n" is invalid`

You have updated the PR name, required is to update commit message.